utils/validate_meta_files.py

```diff
@@ -2,34 +2,64 @@
 
 
 def main():
-    asset_path = "Project/Assets"
+    asset_paths = [
```

Comment:
Just copied from master. This would have caught the fact that I forgot Analytics.meta file.